apis/infrastructure/v1/nim_types.go

```diff
@@ -0,0 +1,12 @@
+package v1
```

Comment:
this should have been moved to kserver_type.go

Reply:
probably you need rebase your "for" branch first

Reply:
I think I just missed removing this file